bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -87,7 +87,56 @@ def __init__(self,
         
         # -- keeps track of gradients applied
         self.backward_gradients = 0 
-        
+        self.set_fine_tuning_params()
+
+    def set_fine_tuning_params(self):
+        r''' Set to tune only the parameter of the last layer
+        '''
+        def find_last_layer(model):    
```

Comment:
Could we also have type hinting here for what's returned?

Reply:
sure! my bad haha